agnostic-svelte-ts/.storybook/yourTheme.js

```diff
@@ -0,0 +1,39 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+	base: 'light',
+
+	// blue primary
```

Comment:
nice touch!

Reply:
Thanks but I just brought over the config from the svelte project 😄